pydruid/client.py

```diff
@@ -386,11 +403,20 @@ class PyDruid(BaseDruidClient):
     def __init__(self, url, endpoint):
         super(PyDruid, self).__init__(url, endpoint)
 
+    def ssl_context(self):
+        ctx = ssl.create_default_context()
+        ctx.check_hostname = False
+        ctx.verify_mode = ssl.CERT_NONE
+        return ctx
+
     def _post(self, query):
         try:
             headers, querystr, url = self._prepare_url_headers_and_body(query)
             req = urllib.request.Request(url, querystr, headers)
-            res = urllib.request.urlopen(req)
+            if self.ignore_certificate_errors:
```

Comment:
If you do the suggestion above, this would just be `res = urllib.request.urlopen(req, context=self.ssl_context())` rather than having a conditional.